index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    rel="stylesheet">
+</head>
+<body>
+    <div class="main-container">
+        <nav class="nav-bars">
+            <nav class="logo">
+              <h2>DNG</h2>
+            </nav>
```

Comment:
- [ ] good job for wrapping the hamburger menu in a nav tag but it isn't a good idea to wrapp the logo in a nav tag because it is not a navigation element.
- [ ] Please the logo should be a link that sends the user to the home page if you check websites like youtube or Facebook you will notice that they provide a link when the user click on the logo which sends you to the home page it would be better to provide such functionality.